pkg/integration/utils.go

```diff
@@ -36,9 +46,8 @@ func ProcessExitCode(err error) (exitCode int) {
 	if err != nil {
 		var exiterr error
 		if exitCode, exiterr = GetExitCode(err); exiterr != nil {
-			// TODO: Fix this so we check the error's text.
-			// we've failed to retrieve exit code, so we set it to 127
-			exitCode = 127
+			// we've failed to retrieve exit code, so we set it to 125
+			exitCode = 125
```

Comment:
Might as well just make GetExitCode() return 125 in these cases instead of `exitCode, fmt.Errorf("failed to get....")` - then we can have it just return 'exitCode' w/o the error.  No one else calls GetExitCode() except right here anyway.